test/user.e2e-spec.ts

```diff
@@ -27,13 +27,15 @@ describe('Users API Testing (e2e)', () => {
         UserModule,
         TypeOrmModule.forRootAsync({
           imports: [ConfigModule],
-          useFactory: (config: ConfigService) => ({
+          useFactory: () => ({
             type: 'postgres',
-            host: 'localhost',
+            host: process.env.TEST_DATABASE_HOST,
+            username: process.env.TEST_DATABASE_USER,
             port: 5432,
-            database: 'open_forum_test_db',
-            autoLoadEntities: true,
+            database: 'postgres',
+            password: 'postgres',
```

Comment:
Initially, all the values were hard-coded since we were only testing the database locally. My original thinking behind this was to only provide env variables to the configuration where I have to change for the CI tests, but now that I think of it, I should have some standardization and use .env variables everywhere. It was suggested that maybe I should just hardcode the configuration, but that was before I wrote the CI integration.